aea/cli/__main__.py

```diff
@@ -48,9 +49,13 @@
 @click.version_option('0.1.0')
 @click.pass_context
 @click_log.simple_verbosity_option(logger, default="INFO")
-def cli(ctx) -> None:
```

Comment:
@marcofavorito I am not happy with this as the solution as it makes the cli less succinct. I think we should avoid the args to the cli and handle this via the config file as well. The config file can provide a default path to `temp/logs.txt` and so no further config is needed.

Reply:
agreed!